fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBlockTags.java

```diff
@@ -166,6 +195,46 @@ private ConventionalBlockTags() {
 	 */
 	public static final TagKey<Block> HIDDEN_FROM_RECIPE_VIEWERS = register("hidden_from_recipe_viewers");
 
+	/**
+	 * Blocks which are often replaced by deepslate ores, i.e. the ores in the tag {@link #ORES_IN_GROUND_DEEPSLATE}, during world generation
```

Comment:
Question: Why is a sperate tag needed for these?

Reply:
This is more of the actual ground block itself (deepslate, stone, netherrack) rather than the ores themselves. Twilight Forest uses these tags for their Ore Magnet. Not sure how but I can ask them to come comment here

Reply:
The original source of the ore_bearing_ground tags came from here. Pinging @SuperMartijn642 if they want to elaborate more about it. Does seem like mods make use of it. Not many but some do.
![image](https://github.com/user-attachments/assets/3cfe49a1-fb64-4f2b-9f9f-a6fac44cb6b7)
https://github.com/MinecraftForge/MinecraftForge/pull/7891#issuecomment-908245244

Reply:
The reasoning behind these tags was that some mods generated ore related recipes from the ore tags and that some machines, for example the pulverizer from Thermal Expansion, may drop the block the ore is in (so stone up to 1.17) as a byproduct.
In 1.17, deepslate variants of ores were added to vanilla, hence there was a need to differentiate between ores in stone and ores in deepslate. As there are mods which add additional ore bearing blocks, the idea was that if there's a way to identify the ground block the ore is in, recipes for those ores could be generated automatically as well.

So, for example, a mod may add a marble block and a marble coal ore block.
Then the marble coal ore block would go in `ore_in_ground/marble` and the marble block would go in `ore_bearing_ground/marble`.
Some mod which may want to find all coal ores and their corresponding ground block could then get all coal ore blocks from `ores/coal`. Then for each, check in which `ore_in_ground/x` tag the ore block is and then obtain the ground block from `ore_bearing_ground/x`.

I personally haven't used these tags and I am not sure whether these tags have actually been adopted by mods since they were added to NeoForge.

Reply:
I have an item that I have yet to port for my embers rekindled port called metallurgic dust.
When used on an ore vein it transmutes the entire vein into a random different ore from the same stone type, each ore block also has a chance of just turning into stone.
Having these tags would be great for making this item work out of the box with modded ores.

Reply:
> Twilight Forest uses these tags for their Ore Magnet. Not sure how they use it but I can ask them to come comment here

For clarity, it is used for procedurally determining the block that we replace at an Ore's location when translocating the Ore block via the Ore Magnet / Tree of Mining. Thanks to the tags, there is zero hardcoding for the relationships between "source blocks" and the actual ore blocks